incubator/kong/README.md

```diff
@@ -0,0 +1,142 @@
+# Kong
+
+[Kong](https://getkong.org/) is a scalable, open source API Layer (also known as an API Gateway, 
+or API Middleware). Kong runs in front of any RESTful API and is extended through Plugins, which 
+provide extra functionality and services beyond the core platform. See 
+[What is Kong?](https://getkong.org/about/) for more information.
+
+__DISCLAIMER:__ This Helm Chart was created using Kong documentation, but it is not an officially 
+supported or maintained Mashape product.
+
+## Introduction
+
+This Chart bootstraps Kong as a Kubernetes deployment with support for proxy and admin endpoints.
+
+__NOTE:__ This Chart can be configured to use Kong supported data stores of PostgreSQL or Cassandra
+but it does not intend to manage or deploy those sources. See the 
+[PostgreSQL Chart](https://github.com/kubernetes/charts/tree/master/stable/postgresql) 
```

Comment:
Why not install this as a dependent chart so that this works out of the box? As is this chart is not useful without deploying more things. For more info:
https://github.com/kubernetes/helm/blob/df92425fa9d06f1233876871e96e4d82c194215b/docs/chart_best_practices/requirements.md